### PR TITLE
Feat/block indexer golden age

### PR DESCRIPTION
**Pull request by leonimella**
- Adds new block information to the indexer, DB and API

### CODE REVIEW

File: apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transactions.ex [hunk 1]
```diff
@@ -153,6 +153,7 @@ defmodule EthereumJSONRPC.Transactions do
   """
   def to_elixir(transactions, block_timestamp \\ nil) when is_list(transactions) do
     transactions
+    |> Enum.filter(fn tx -> tx["type"] != "0x2" end)
     |> Enum.map(&Transaction.to_elixir(&1, block_timestamp))
```
Comment (evseevnn) [inline apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transactions.ex:157]: We don't need to filter all transactions by type 0x2, we need to define transactions of type 0x2 and store additional fields for them. We may need to display them as a separate list in the interface, so we may need the ability to filter them in the API part.

File: apps/explorer/lib/explorer/chain/block.ex [hunk 2]
```diff
@@ -24,6 +24,18 @@ defmodule Explorer.Chain.Block.Schema do
...
+                              field(:etx_set_hash, Hash.Full)
+                              field(:parent_uncled_sub_delta_s, {:array, :binary})
+                              field(:efficiency_score, :binary)
+                              field(:threshold_count, :binary)
+                              field(:expansion_number, :binary)
+                              field(:etx_eligible_slices, Hash.Full)
+                              field(:prime_terminus, Hash.Full)
+                              field(:interlink_root_hash, Hash.Full)
+                              field(:uncled_s, :binary)
+                              field(:interlink_hashes, {:array, Hash.Full})
```
Comment (evseevnn) [inline apps/explorer/lib/explorer/chain/block.ex:38]: Not sure, but maybe will good idea to keep all work object in db and add label with name of parent field near value, for example label [WorkObject] 0x2123

File: apps/explorer/lib/explorer/chain/block.ex [hunk 3]
```diff
@@ -122,15 +134,15 @@ defmodule Explorer.Chain.Block do
 
   use Explorer.Schema
 
-  alias Explorer.Chain.{Block, Hash, Transaction, ExternalTransaction, Wei}
+  alias Explorer.Chain.{Block, Hash, Transaction, Wei}
```
Comment (evseevnn) [inline apps/explorer/lib/explorer/chain/block.ex:137]: Yes, if I understand correctly currently we will determine external transaction by markers in transaction hash. So, looks like we no need keep external transaction separately.

**Review by evseevnn — CHANGES_REQUESTED** [review]

File: apps/explorer/lib/explorer/chain/block.ex (hunk 2 above)
Comment (leonimella) [reply apps/explorer/lib/explorer/chain/block.ex:38]: Could you elaborate a bit more on this suggestion? I didn't get what is the improvement here.

**Comment by leonimella** [comment]
Implementation moved to #3

**Pull request closed without merging**